nautobot/core/tests/nautobot_config.py

```diff
@@ -45,12 +45,12 @@ def is_truthy(arg):
 
 # Redis variables
 REDIS_HOST = os.getenv("REDIS_HOST", "localhost")
-REDIS_PORT = os.getenv("REDIS_PORT", 6379)
+REDIS_PORT = os.getenv("REDIS_PORT", "6379")
```

Comment:
Just checking, does this need to be cast to `int` for safety?

Reply:
This isn't necessary. Django handles it internally.